includes/core/classes/class-event-feed.php

```diff
@@ -0,0 +1,259 @@
+
+		$event = new Event( get_the_ID() );
+		$venue = $event->get_venue_information();
+
+		$event_info = $this->get_event_datetime_info( $event );
+
+		// Add venue information.
+		if ( $venue && ! empty( $venue['name'] ) ) {
+			$event_info[] = sprintf(
+				/* translators: %s: Venue name */
+				__( 'Venue: %s', 'gatherpress' ),
+				$venue['name']
+			);
+		}
+
+		// Build the customized excerpt.
+		$custom_excerpt = '';
+
+		if ( ! empty( $event_info ) ) {
+			$custom_excerpt .= '<p><strong>' . implode( ' | ', $event_info ) . '</strong></p>';
+		}
+
+		// Add the original excerpt if it exists and is different from the content.
+		if ( ! empty( $excerpt ) && get_the_content() !== $excerpt ) {
+			$clean_excerpt = wp_strip_all_tags( $excerpt );
+			$clean_excerpt = preg_replace( '/\s+/', ' ', $clean_excerpt ); // Normalize whitespace.
+			$clean_excerpt = trim( $clean_excerpt );
+
+			if ( ! empty( $clean_excerpt ) ) {
+				$custom_excerpt .= '<p>' . $clean_excerpt . '</p>';
+			}
+		}
+
+		return $custom_excerpt;
+	}
+
+	/**
+	 * Customize RSS content for events to include event details.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $content The current content.
+	 * @return string The customized content.
+	 */
+	public function customize_event_content( string $content ): string {
+		global $post;
+
+		// Only apply to events.
+		if ( Event::POST_TYPE !== get_post_type( $post ) ) {
+			return $content;
+		}
+
+		$event = new Event( get_the_ID() );
+		$venue = $event->get_venue_information();
+
+		$event_info = $this->get_event_datetime_info( $event );
+
+		// Add venue information.
+		if ( $venue && ! empty( $venue['name'] ) ) {
+			$event_info[] = sprintf(
+				/* translators: %s: Venue name */
+				__( 'Venue: %s', 'gatherpress' ),
+				$venue['name']
+			);
+		}
+
+		// Build the customized content.
+		$custom_content = '';
+
+		if ( ! empty( $event_info ) ) {
+			$custom_content .= '<p><strong>' . implode( ' | ', $event_info ) . '</strong></p>';
+		}
+
+		// For RSS feeds, provide a cleaner version of the content.
+		// Strip out complex HTML and keep only essential information.
+		$clean_content = wp_strip_all_tags( $content );
+		$clean_content = preg_replace( '/\s+/', ' ', $clean_content ); // Normalize whitespace.
+		$clean_content = trim( $clean_content );
+
+		if ( ! empty( $clean_content ) ) {
+			$custom_content .= '<p>' . $clean_content . '</p>';
+		}
+
+		// Add comments section.
+		$comments = get_comments(
+			array(
+				'post_id' => get_the_ID(),
+				'status'  => 'approve',
+				'number'  => 5, // Limit to 5 most recent comments.
+			)
+		);
+
+		if ( ! empty( $comments ) ) {
+			$custom_content .= '<h3>' . __( 'Comments:', 'gatherpress' ) . '</h3>';
+			foreach ( $comments as $comment ) {
+				$custom_content .= '<p><strong>' . esc_html( $comment->comment_author ) . ':</strong> ';
+				$custom_content .= esc_html( wp_strip_all_tags( $comment->comment_content ) ) . '</p>';
+			}
+		}
+
+		return $custom_content;
+	}
+
+	/**
```

Comment:
What is the purpose of that method?

It seems _(to me)_ to replicate, what `register_post_type()` is already doing?

If there is a real reason for having it, it might be better to build the URL based on the `gatherpress_general['urls']['events'] `setting and not with the hardcoded 'events' string.

Reply:
@carstingaxion I am now using settings for building the url. The purpose of the method is to customize output of the event content for the purposes of RSS feed.

Reply:
Hi @jmarx,

I'm sorry that I need to ask again: _Why are you creating that new rewrite_rule?_
I don't get it. 

**Before:**
<img width="1976" height="836" alt="image" src="https://github.com/user-attachments/assets/7200d68b-4449-48ee-9f7c-988eaabd546a" />

**After:**
<img width="1976" height="898" alt="image" src="https://github.com/user-attachments/assets/ef7ae230-27d4-4c6c-a073-d6e5c75941cf" />

Reply:
@carstingaxion 
Sorry, I completely misunderstood your comment/question. I get what you're saying now. I removed the redundant rewrite rule.